happybase/client.py

```diff
@@ -1,25 +1,40 @@
 # -*- coding: utf-8 -*-
+
+import time
+import logging
+import datetime
+from functools import partial
+from socket import error as socket_error
+from collections import deque, defaultdict
+
+import numpy as np
 from thriftpy.thrift import TClient
 from thriftpy.thrift import TApplicationException
 from thriftpy.transport import TTransportException
-from socket import error as socket_error
-from collections import deque
-from time import sleep, time
-import logging
-import numpy as np
+
 
 logger = logging.getLogger(__name__)
 
 
+TIMINGS_KEEP_LIMIT = 50
```

Comment:
Increased up to 200 per AP entry, should be enough, as we're going to send stats per minute for now.